tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -691,8 +737,9 @@ TEST_CASE("Test collective messaging locally and across hosts", "[mpi]")
 
         // Check the host that the root is on
         for (int rank : remoteWorldRanks) {
-            if (rank == remoteRankB)
+            if (rank == remoteRankB) {
```

Comment:
I am surprised that this wasn't brought up before by `clang-tidy` I'll note down to look into it.